core/client/batch_get_and_verify.go

```diff
@@ -95,7 +96,8 @@ func (c *Client) BatchVerifyGetUserIndex(ctx context.Context, userIDs []string)
 }
 
 // BatchVerifiedGetUser returns verified leaf values by userID.
-func (c *Client) BatchVerifiedGetUser(ctx context.Context, userIDs []string) (map[string]*pb.MapLeaf, error) {
+func (c *Client) BatchVerifiedGetUser(ctx context.Context, userIDs []string) (
```

Comment:
The doc string should be updated to explain the relationship between the returned values. 

It's not immediately clear to me what the contract of this method is. Is it supposed to return the latest value from the server, or is any "recent" (or even historical) revision okay? Passing in an explicit revision for the map would avoid this, but may go against some other design principles here?

Reply:
The API contract is indeed unclear and unverified.  
I'll add a TODO to fix this since the change affects multiple files. 
The intent is that this would fetch values against the *latest* map root, but as it is, the client does not enforce any constraint on the freshness of the MapRoot.